docs/source/tools/profiler.rst

```diff
@@ -22,13 +22,21 @@ assistant. We first create the project and build the DAR.
   cd profile-tutorial
   daml build
 
-Next we load the DAR into Sandbox with a special ``--profile-dir``
+Next we load the DAR into Sandbox with a special ``profile-dir``
 option. Sandbox will behave as usual but all profile results will be
-written to that directory.
+written to that directory. For this, we first create a configuration
+file that sets the ``profile-dir`` for Sandbox:
+
+.. code-block:: none
+   :caption: profile.conf
+
+   canton.participants.sandbox.features.profile-dir = profile-results
```

Comment:
we may want a CLI option here eventually but for now this seems good enough and it works (tested).